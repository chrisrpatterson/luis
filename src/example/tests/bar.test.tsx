import * as React from 'react';
import * as renderer from 'react-test-renderer';

describe('Component', () => {
  describe('Doo', function() {
    it('tests', function() {});

    describe('Bar', () => {
      function component() {
        return <div>Bar Component 123</div>;
      }

      it('fails', function() {
        throw new Error('Failed miserably');
      });

      it('renders', function() {
        expect(renderer.create(component())).toMatchSnapshot();
      });

      return {
        component: component()
      };
    });
  });
});
