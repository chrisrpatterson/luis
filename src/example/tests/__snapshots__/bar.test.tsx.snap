// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Doo Bar renders 1`] = `
<div>
  Bar Component
</div>
`;

exports[`Component Doo Bar renders1 with really long name there it is 1`] = `
<div>
  Bar Component
</div>
`;
